configs/config.s10_2035_flexible_EGS_BBB.yaml

```
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+#
+# SPDX-License-Identifier: CC0-1.0
+
+version: 0.3.0
+tutorial: false
+
+logging:
+  level: INFO
+  format: "%(levelname)s:%(name)s:%(message)s"
+
+countries: ["US"]
+
+# Options specific to geothermal project
+US:
+  # Config options to scale demand to future years based on NREL EFS study
+  demand_projection:
+    planning_horizon: 2035 # Year to be considered for future demand projections between 2030-2050
+    scenario: "Reference" # Choices - [Reference, Moderate, High]
+    thermal_scenario: "reference case"
+
+
+run:
+  name: "US_2035_BBB" # use this to keep track of runs with different settings
+  sector_name: "US_2025_BBB"
```

Best to keep same name for both a power-only and cross-sectoral runs.
```suggestion
  sector_name: "US_2055_BBB"
```